subsys/mpsl/clock_ctrl/Kconfig

```diff
@@ -13,3 +13,23 @@ config MPSL_USE_EXTERNAL_CLOCK_CONTROL
 	help
 	  This option configures MPSL to use an external clock driver, and
 	  not the clock driver provided as part of the MPSL library.
+
+if MPSL_USE_EXTERNAL_CLOCK_CONTROL
+
+config MPSL_EXT_CLK_CTRL_CLOCK_REQUEST_WAIT_TIMEOUT_MS
+	int ""
+	default CLOCK_CONTROL_NRF2_NRFS_CLOCK_TIMEOUT_MS if CLOCK_CONTROL_NRF2
+	default 1000
```

Comment:
there is a prompt but the prompt is empty, this is not right. If this should not be user adjustable, drop the prompt